<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewProject</class>
 <widget class="QDialog" name="DialogNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New project</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>untitled</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelColorSystem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color system:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonColorSystemNtsc">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NTSC</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupColorSystem</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonColorSystemPal">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PAL</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupColorSystem</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonColorSystemSecam">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SECAM</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupColorSystem</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="labelScanlines">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scanlines:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxScanlines">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>192</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonModeAsymmetric">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Asymmetric</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupMode</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonModeMirror">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupMode</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonModeSymmetric">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Symmetric</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroupMode</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupColorSystem"/>
  <buttongroup name="buttonGroupMode"/>
 </buttongroups>
</ui>
